routers/repo/view.go

```diff
@@ -396,6 +396,20 @@ func renderFile(ctx *context.Context, entry *git.TreeEntry, treeLink, rawLink st
 	isLFSFile := false
 	ctx.Data["IsTextFile"] = isTextFile
 
+	isDisplayingSource := ctx.Query("display") == "source"
+	ctx.Data["IsDisplayingSource"] = isDisplayingSource
+	isDisplayingRendered := !isDisplayingSource
+	ctx.Data["IsDisplayingRendered"] = isDisplayingRendered
+	isRepresentableAsText := base.IsRepresentableAsText(buf)
+	ctx.Data["IsRepresentableAsText"] = isRepresentableAsText
+	if !isRepresentableAsText {
```

Comment:
So we should also change `ctx.Data["IsDisplayingSource"]`?

Reply:
Fixed.